task8/TutterApp/src/main/java/dao/PostDAO.java

```diff
@@ -0,0 +1,15 @@
+package dao;
+
+import forms.FilterForm;
+import models.Post;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface PostDAO {
+    Optional<Post> get(Long id);
```

Comment:
Please use primitive numerics where possible. `Long` is a boxed (objectified) version of `long`, it does not bring any benefit except being suitable for use in generics